requirements.txt

```diff
@@ -2,12 +2,11 @@
 # on Mozilla WebQA projects
 
 PyYAML==3.10
-execnet==1.0.9
 certifi==0.0.8
 chardet==1.0.1
-py==1.4.26
-pytest==2.7.0
-pytest-mozwebqa
+py==1.4.29
+pytest==2.7.2
+pytest-selenium
 requests==2.4.3
 selenium
 bidpom
```

Comment:
This seems to be causing a problem. I believe we need to use the package `browserid` rather than `bidpom` until the suite is upgraded to work with bidpom 2.0. It's definitely generating errors for me when I run it locally.